cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc

```diff
@@ -442,6 +459,112 @@ Replaced by:
 ----
 WHERE NOT isEmpty([1, 2, 3])
 ----
+
+a|
+label:functionality[]
+label:removed[]
+[source, cypher, role="noheader"]
+----
+distance(n.prop, point({x:0, y:0})
+----
+a|
+Replaced by:
+[source, cypher, role="noheader"]
+----
+point.distance(n.prop, point({x:0, y:0})
+----
+
+a|
+label:functionality[]
+label:removed[]
+[source, cypher, role="noheader"]
+----
+point({x:0, y:0}) <= point({x:1, y:1}) <= point({x:2, y:2})
+----
+a|
+Using inequality operators `<`, `<=`, `>`, and `>=` on spatial points is removed.
```

Comment:
```suggestion
The ability to use the inequality operators `<`, `<=`, `>`, and `>=` on spatial points was removed.
```

Reply:
It looks like all other places in this page are using 'is removed' and 'is deprecated' rather than 'was' so if you want to change that, it would be better to do it as a separate task for the whole page. I will change the first part though